app/screens/home/channel_list/categories_list/header/header.test.tsx

```diff
@@ -17,6 +17,7 @@ describe('components/channel_list/header', () => {
                 canJoinChannels={true}
                 canInvitePeople={true}
                 displayName={'Test!'}
+                pushDisabledAck={true}
```

Comment:
Do we want to add test for `pushDisabledAck={false}`? If I am not mistaken there are no test for pushProxyStatus in general, so those could be welcomed also.

Reply:
sure!